testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -4043,3 +4043,31 @@ QUnit.testInActiveWindow("Searching should work correctly in grouped tagBox (T51
     assert.equal($tagContainer.find("." + TAGBOX_TAG_CONTENT_CLASS).length, 2, "selected tags rendered");
     assert.equal($.trim($tagContainer.text()), "Item1Item3", "selected values are rendered");
 });
+
+QUnit.test("Items is not selected when values is set on the onSelectAllValueChanged event", function(assert) {
+    var dataSource = ["Test 1", "Test 2", "Test 3", "Test 4"];
+
+    $("#tagBox").dxTagBox({
+        opened: true,
+        dataSource: {
+            paginate: true,
+            pageSize: 2,
+            requireTotalCount: true,
```

Comment:
requireTotalCount is necessary only for "allPages" mode AFAIK

Reply:
The requireTotalCount is unnecessary for this test. I will remove this, in the following commit.